homeassistant/components/prusalink/strings.json

```diff
@@ -25,6 +25,40 @@
           "pausing": "Pausing",
           "printing": "Printing"
         }
+      },
+      "heatbed_temperature": {
+        "name": "Heatbed temperature"
```

Comment:
You need to adjust the tests accordingly => `sensor.mock_title_heatbed` versus `sensor.mock_title_heatbed_temperature`